$OpenBSD: patch-scheduler_ipp_c,v 1.13 2013/10/27 08:50:17 ajacoutot Exp $

Increase timeout for the PPD file to be copied so that it doesn't fail
on low-end configurations.

--- scheduler/ipp.c.orig	Fri Sep 27 16:23:42 2013
+++ scheduler/ipp.c	Sun Oct 27 09:01:11 2013
@@ -4563,7 +4563,7 @@ copy_model(cupsd_client_t *con,		/* I - Client connect
   close(temppipe[1]);
 
  /*
-  * Wait up to 30 seconds for the PPD file to be copied...
+  * Wait up to 90 seconds for the PPD file to be copied...
   */
 
   total = 0;
@@ -4583,7 +4583,7 @@ copy_model(cupsd_client_t *con,		/* I - Client connect
     FD_SET(temppipe[0], &input);
     FD_SET(CGIPipes[0], &input);
 
-    timeout.tv_sec  = 30;
+    timeout.tv_sec  = 90;
     timeout.tv_usec = 0;
 
     if ((i = select(maxfd, &input, NULL, NULL, &timeout)) < 0)
